lib/fly.js

```diff
@@ -85,12 +85,18 @@ module.exports = Fly
  * @return                      The current Fly instance.
  */
 Fly.prototype.source = function (globs) {
-	globs = arrify(globs)
+	globs = flatten([].slice.call(arguments))
```

Comment:
The line that I took out `arrify` was only being passed in `globs` which is the first argument so if you had `this.source('somepath')` or `this.source(['somepath'])` both would resolve to be `['somepath]'` the docs say that `source` takes `...globs` which is `this.source('somepath', 'someotherpath')` and if the `arrify` version is used then `'someotherpath'` is ignored. If you try to pass in `arguments` to `arrify` it just returns the arguments object as an array.